msal/src/Microsoft.Identity.Client/Exceptions/MsalErrorMessage.cs

```diff
@@ -70,7 +70,7 @@ internal static class MsalErrorMessage
         public const string MissingAuthenticateHeader = "WWW-Authenticate header was expected in the response";
 
         public const string MultipleTokensMatched =
-            "The cache contains multiple tokens satisfying the requirements. Call AcquireToken again providing more requirements like authority";
+            "The cache contains multiple tokens satisfying the requirements. Try to clear token cache and report the issue";
```

Comment:
>and report the issue [](start = 102, length = 20)

perhaps the above also needs to be there. or the clear cache. I would not like to see a report the issue in this text.

Reply:
consider adding an aka.ms/msal-multipletokensfoundincache

---
In reply to: [216107714](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1246#discussion_r216107714) [](ancestors = 216107714)